sdk/keyvault/platform-matrix.json

```diff
@@ -28,7 +28,21 @@
       },
       "ArmTemplateParameters": "@{ enableHsm = $true }",
       "AZURE_TEST_HTTP_CLIENTS": "netty",
-      "JavaTestVersion": "1.11",
+      "JavaTestVersion": "1.8",
```

Comment:
Is this intentional to drop the version from 11 to 8?

Reply:
Yes. I own this service and wanted to test on Managed HSM (a more secure version of Key Vault) on the lowest and highest supported versions of Java. Now that I think about it, should we do it on every version?